nifi-asymmetrik-standard-bundle/nifi-asymmetrik-standard-reporting-tasks/src/main/java/com/asymmetrik/nifi/reporting/AbstractNiFiClusterMetricsReporter.java

```diff
@@ -376,26 +379,21 @@ private void populateOutputPortSnapshots(ProcessGroupStatus status, SystemMetric
         return metrics;
     }
 
-    /**
-     * Converts IP address specified as byte[] to dot notation String equivalent
-     *
-     * @return String representation of ipAddress
-     */
-    public static String getIpAddress() {
-        try {
-            byte[] address = InetAddress.getLocalHost().getAddress();
-            if (address == null) {
-                return "";
+    public String getHostname() {
```

Comment:
Since these are AWS specific, none of these changes should be in the abstract class. It may be better to subclass this abstract class with another AWS-specific abstract class.